packs/data/pathfinder-bestiary-3.db/namorrodor.json

```
@@ -320,7 +320,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>@Template[type:emanation|distance:30]{30 feet} @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<hr />\n<p>Wind whistles through the namorrodor's jutting bones with an eerie tone. A creature entering or beginning its turn in the area must attempt a @Check[type:will|dc:19] save, becoming @UUID[Compendium.pf2e.conditionitems.Frightened]{Frightened 1} on a failure (@UUID[Compendium.pf2e.conditionitems.Frightened]{Frightened 2}on a critical failure). A creature can't reduce its frightened condition below 1 as long as it's in the aura.</p>"
+                    "value": "<p>@Template[type:emanation|distance:30]{30 feet} @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<hr />\n<p>Wind whistles through the namorrodor's jutting bones with an eerie tone. A creature entering or beginning its turn in the area must attempt a @Check[type:will|dc:19] save, becoming @UUID[Compendium.pf2e.conditionitems.Frightened]{Frightened 1} on a failure (@UUID[Compendium.pf2e.conditionitems.Frightened]{Frightened 2} on a critical failure).</p>\n<p>A creature can't reduce its frightened condition below 1 as long as it's in the aura.</p>"
```

```suggestion
                    "value": "<p>@Template[type:emanation|distance:30]{30 feet} @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<hr />\n<p>Wind whistles through the namorrodor's jutting bones with an eerie tone. A creature entering or beginning its turn in the area must attempt a @Check[type:will|dc:19] save, becoming @UUID[Compendium.pf2e.conditionitems.Frightened]{Frightened 1} on a failure (@UUID[Compendium.pf2e.conditionitems.Frightened]{Frightened 2} on a critical failure). A creature can't reduce its frightened condition below 1 as long as it's in the aura.</p>"
```
That clause isn't a new paragraph in the book.